posts/views.py

```diff
@@ -52,6 +52,12 @@ def get(self, request, pk: int):
             "dislike_count": LikeDislike.objects.filter(
                 post_id=pk, is_like=False
             ).count(),
+            "like": LikeDislike.objects.filter(
+                post_id=pk, user=request.user, is_like=True
+            ).exists(),
+            "dislike": LikeDislike.objects.filter(
```

Comment:
Not sure, but probably smth like `has_disliked` will be clearer

Reply:
Changed to has_disliked